.gitignore

```diff
@@ -14,3 +14,5 @@ CMakeSettings.json
 
 # clangd
 /.cache/
+CMakeFiles/
+src/CMakeFiles/
```

Comment:
Why this change?

Reply:
To avoid in source tree build generated files being listed in git add -A and git commit -a

Reply:
If you create a folder named "build*", they should already be ignored. For example:
```
cd woof
cmake -B build -G Ninja
```

Reply:
Indeed, this change is unrelated.